packages/server/modules/fileuploads/repositories/fileUploads.ts

```diff
@@ -75,60 +75,68 @@ export const saveUploadFileFactory =
     return newRecord as FileUploadRecord
   }
 
-const getPendingUploadsBaseQuery = (
-  streamId: string,
-  options?: Partial<{ ignoreOld: boolean; limit: number }>
-) => {
-  const { ignoreOld = true, limit } = options || {}
+const getPendingUploadsBaseQueryFactory =
```

Comment:
this is the "main one", but I also updated the 2 downstream repo functions that use it